---
permalink: /
title: "About me"
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

Leiting Li gained his B.S. (2010) and Ph.D. (2016) in Nanjing Agricultural University. Afterwards, he was a postdoc in CAS Center for Excellence in Molecular Plant Sciences (CEMPS) from 2017 to 2022. Now, he is a research assistant in Lingang Laboratory. He is mainly working on genomics and bioinformatics. 

## Education

-	2006.09 ~ 2010.06, B.S., Nanjing Agricultural University 
-	2010.09 ~ 2016.12, Ph.D., Nanjing Agricultural University
-	2011.06 ~ 2012.07, Visiting student, University of Illinois at Urbana-Champaign	

## Academic experience

-	2017.01 ~ 2022.12, Postdoc, Shanghai Center for Plant Stress Biology, CAS Center for Excellence in Molecular Plant Sciences
-	2022.12 ~ Present, Research Assistant, Lingang Laboratory

## Academic funding

-	2019.08, Awarded an National Natural Science Foundation of China (NSFC) Youth Program (CNY 240,000) entitled “The origin and evolution of broomcorn millet based on whole-genome resequencing” (31900189)

## Publications

**(First/co-first authored publications)**

1.	**Li, L.**, Song, J., Zhang, M., Iqbal, S., Li, Y., Zhang, H., and Zhang, H. (2023). A near complete genome assembly of chia assists in identification of key fatty acid desaturases in developing seeds. Front in Plant Sci, 2022.08.15.504044.
2.	**Li, L.**<sup>#</sup>, Li, M.<sup>#</sup>, Wu, J., Yin, H., Dunwell, J.M., and Zhang, S. (2022). Genome-wide identification and comparative evolutionary analysis of sorbitol metabolism pathway genes in four Rosaceae species and three model plants. BMC Plant Biol 22:341.
3.	**Li, L.**, and Zhang, H. (2022). The broomcorn millet genome. In: Underutilised Crop Genomes--Chapman, M.A., ed. Switzerland: Springer Nature. 1-17. (In publication)
4.	**Li, L.**, Deng, C.H., Knabel, M., Chagne, D., Kumar, S., Sun, J., Zhang, S., and Wu, J. (2017). Integrated high-density consensus genetic map of Pyrus and anchoring of the 'Bartlett' v1.0 (Pyrus communis) genome. DNA Res 24:289-301.
5.	Qin, M.F. <sup>#</sup>, **Li, L.T.** <sup>#</sup>, Singh, J., Sun, M.Y., Bai, B., Li, S.W., Ni, J.P., Zhang, J.Y., Zhang, X., Wei, W.L., et al. (2022). Construction of a high-density bin-map and identification of fruit quality-related quantitative trait loci and functional genes in pear. Hortic Res 9:uhac141.
6.	Richards, S.M. <sup>#</sup>, **Li, L.** <sup>#</sup>, Breen, J., Hovhannisyan, N., Estrada, O., Gasparyan, B., Gilliham, M., Smith, A., Cooper, A., and Zhang, H. (2022). Recovery of chloroplast genomes from medieval millet grains excavated from the Areni-1 cave in southern Armenia. Sci Rep 12:15164.
7.	Kiani-Pouya, A. <sup>#</sup>, **Li, L.** <sup>#</sup>, Rasouli, F., Zhang, Z., Chen, J., Yu, M., Tahir, A., Hedrich, R., Shabala, S., and Zhang, H. (2022). Transcriptome analyses of quinoa leaves revealed critical function of epidermal bladder cells in salt stress acclimation. Plant Stress:100061.
8.	Sun, J. <sup>#</sup>, **Li, L.** <sup>#</sup>, Wang, P., Zhang, S., and Wu, J. (2017). Genome-wide characterization, evolution, and expression analysis of the leucine-rich repeat receptor-like protein kinase (LRR-RLK) gene family in Rosaceae genomes. BMC Genomics 18:763.
9.	Chen, H.<sup>#</sup>, Song, Y. <sup>#</sup>, **Li, L.-T.** <sup>#</sup>, Khan, M.A., Li, X.-G., Korban, S.S., Wu, J., and Zhang, S.-L. (2015). Construction of a high-density simple sequence repeat consensus genetic map for pear (Pyrus spp.). Plant Molecular Biology Reporter 33:316-325.
10.	Wu, J. <sup>#</sup>, **Li, L.T.** <sup>#</sup>, Li, M., Khan, M.A., Li, X.G., Chen, H., Yin, H., and Zhang, S.L. (2014). High-density genetic linkage map construction and identification of fruit-related QTLs in pear using SNP and SSR markers. J Exp Bot 65:5771-5781.

**(Selected co-authored publications)**

11.	Zhu, S., Gu, J., Yao, J., Li, Y., Zhang, Z., Xia, W., Wang, Z., Gui, X., **Li, L.**, Li, D., et al. (2022). Liquid-liquid phase separation of RBGD2/4 is required for heat stress resistance in Arabidopsis. Dev Cell 57:583-597 e586.
12.	Zou, C., **Li, L.**, Miki, D., Li, D., Tang, Q., Xiao, L., Rajput, S., Deng, P., Peng, L., Jia, W., et al. (2019). The genome of broomcorn millet. Nature communications 10:436.
13.	Tang, T., Zhang, Y., Wang, Y., Cai, Z., Lu, Z., **Li, L.**, Huang, R., Hagelkruys, A., Matthias, P., Zhang, H., et al. (2019). HDAC1 and HDAC2 Regulate Intermediate Progenitor Positioning to Safeguard Neocortical Development. Neuron 101:1117-1133 e1115.
14.	Qiao, X., Li, Q., Yin, H., Qi, K., **Li, L.**, Wang, R., Zhang, S., and Paterson, A.H. (2019). Gene duplication and evolution in recurring polyploidization-diploidization cycles in plants. Genome Biol 20:38.
15.	Wu, J., Wang, Y., Xu, J., Korban, S.S., Fei, Z., Tao, S., Ming, R., Tai, S., Khan, A.M., Postman, J.D., … **Li, L.** et al. (2018). Diversification and independent domestication of Asian and European pears. Genome Biol 19:77.
16.	Denoeud, F., Carretero-Paulet, L., Dereeper, A., Droc, G., Guyot, R., Pietrella, M., Zheng, C., Alberti, A., Anthony, F., Aprea, G., …, **Li, L.T.** et al. (2014). The coffee genome provides insight into the convergent evolution of caffeine biosynthesis. Science 345:1181-1184.
17.	Ming, R., VanBuren, R., Liu, Y., Yang, M., Han, Y., **Li, L.T.**, Zhang, Q., Kim, M.J., Schatz, M.C., Campbell, M., et al. (2013). Genome of the long-living sacred lotus (Nelumbo nucifera Gaertn.). Genome Biol 14:R41.
18.	Wu, J., Wang, Z., Shi, Z., Zhang, S., Ming, R., Zhu, S., Khan, M.A., Tao, S., Korban, S.S., Wang, H., …, **Li. L.**, et al. (2013). The genome of the pear (Pyrus bretschneideri Rehd.). Genome Res 23:396-408.

## Research areas and skills

-	[Omics/bioinformatics] High-throughput sequencing data analysis; whole-genome genome assembly and annotation; whole genome resequencing data analysis; RNA-seq, BS-seq and sRNA-seq data analysis. 
-	[Plant molecular breeding] Construction of plant genetic maps; QTL analysis; GWAS. 
-	[Computer skills] Linux operation system; high-performance computing (HPC) clusters; Shell, Perl and R programming languages.
-	[Data science] Familiar with statistical analysis and data visualization. 

